spec/models/bulkrax/csv_entry_spec.rb

```diff
@@ -0,0 +1,202 @@
+            expect(subject.parsed_metadata['PhysicalInstantiation.annotation_1']).to eq('Test annotation')
+            expect(subject.parsed_metadata['PhysicalInstantiation.holding_organization_1']).to eq('American Archive of Public Broadcasting')
+            expect(subject.parsed_metadata['PhysicalInstantiation_2']).to eq('')
+            expect(subject.parsed_metadata['PhysicalInstantiation.annotation_2']).to eq('Minimal annotation')
+          end
+        end
+      end
+
+      describe 'creates the correct headers' do
+        before do
+          exporter.export
+        end
+
+        context 'including' do
+          it 'valid Asset headers' do
+            expect(headers).to include('Asset_1')
+            expect(headers).to include('Asset.annotation_1')
+            expect(headers).to include('Asset.audience_level_1')
+            expect(headers).to include('Asset.broadcast_date_1')
+            expect(headers).to include('Asset.clip_description_1')
+            expect(headers).to include('Asset.date_1')
+            expect(headers).to include('Asset.episode_description_1')
+            expect(headers).to include('Asset.genre_1')
+            expect(headers).to include('Asset.local_identifier_1')
+            expect(headers).to include('Asset.pbs_nola_code_1')
+            expect(headers).to include('Asset.rights_summary_1')
+            expect(headers).to include('Asset.subject_1')
+            expect(headers).to include('Asset.topics_1')
+          end
+
+          it 'valid PhysicalInstantation headers' do
+            expect(headers).to include('PhysicalInstantiation_1')
+            expect(headers).to include('PhysicalInstantiation.annotation_1')
+            expect(headers).to include('PhysicalInstantiation.date_1')
+            expect(headers).to include('PhysicalInstantiation.holding_organization_1')
+            expect(headers).to include('PhysicalInstantiation.local_instantiation_identifier_1')
+            expect(headers).to include('PhysicalInstantiation.location_1')
+            expect(headers).to include('PhysicalInstantiation.media_type_1')
+            expect(headers).to include('PhysicalInstantiation_2')
+            expect(headers).to include('PhysicalInstantiation.annotation_2')
+            expect(headers).to include('PhysicalInstantiation.location_2')
+            expect(headers).to include('PhysicalInstantiation.media_type_2')
+          end
+        end
+
+        context 'not including' do
+          it 'invalid model headers' do
+            expect(headers).not_to include('Asset.holding_organization_1')
+            expect(headers).not_to include('PhysicalInstantiation.admin_data_gid_1')
+          end
+        end
+      end
+
+      describe 'writes the csv file' do
+        before do
+          exporter.write
+        end
+
+        xit 'with the asset and its children in a single row' do
+          # TODO(alishaevn): write this spec
```

Comment:
@alishaevn can you shed some light on this one?

Reply:
I believe I initially left this as a placeholder to show that a parent asset and it's child works would all be exported on a single row. however, we're not actually creating a csv in this spec and I believe the describe block above for "creates the correct headers" is verification enough.

so this can be removed. suggestion below.